examples/sample/test/Lock.js

```diff
@@ -51,7 +50,7 @@ describe.skip("Lock", function () {
         deployOneYearLockFixture
       );
 
-      expect(await ethers.provider.getBalance(lock.address)).to.equal(
+      expect(await ethers.provider.getBalance(lock.target)).to.equal(
```

Comment:
What's `.target`?

Reply:
This is coming from the new hardhat sample.